realm/src/androidTest/java/io/realm/SortTest.java

```diff
@@ -0,0 +1,202 @@
+/*
```

Comment:
We do have a `NotificationTest.java` which tests functionality and not a class. I extrapolated a naming practice from this example :-)